nle/nethack/nethack.py

```diff
@@ -206,7 +206,13 @@ def __init__(
 
         if options is None:
             options = NETHACKOPTIONS
-        self.options = list(options) + ["name:" + playername]
+
+        self.options = (
```

Comment:
Nit:

```
self.options = list(options)
if options[0][0] != "@":
    self.options.append(f"name:{playername}")
```

and perhaps also a comment what this does.

And perhaps a test to show this works? I'm still a bit unclear if this is a good idea as it pulls in yet another external source of state.

Reply:
Added a comment to the file as well to the README file. 

I think having the config file could potentially help manage NetHack experiments, where a configuration file would be associated with a certain experiment. 

Since the config file also allows changing the visual aspect of NetHack it could also allow for testing transfer and generalization across that modality. 

But yeah, it would be a advanced feature of the NLE, most people would probably not touch this.